backend/response_handler.py

```diff
@@ -0,0 +1,36 @@
+from flask import jsonify, Response
+from json import dumps
+
+class ResponseHandler:
+    """Class to handle consistent JSON responses for the backend."""
+
+    @staticmethod
+    def success(message: str, data: dict = None) ->Response:
+        """Return a consistent JSON response for success."""
+        response = {
+            "status": "success",
+            "message": message,
+            "data": data or {}
+        }
+        return jsonify(response), 200
```

Comment:
method names should be descriptive. Please use `generate_success_response()`, and similar for others. Thinking from the users perspective can help; `ResponseHandler.succes()` vs `ResponseHandler.generate_success_response()`